src/taipy/core/_scheduler/_dispatcher/_standalone_job_dispatcher.py

```diff
@@ -9,22 +9,24 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+import threading
 from concurrent.futures import ProcessPoolExecutor
 from functools import partial
 
 from taipy.config._toml_serializer import _TomlSerializer
 from taipy.config.config import Config
 
 from ...job.job import Job
+from .._abstract_scheduler import _AbstractScheduler
 from ._job_dispatcher import _JobDispatcher
 
 
 class _StandaloneJobDispatcher(_JobDispatcher):
     """Manages job dispatching (instances of `Job^` class) in an asynchronous way using a ProcessPoolExecutor."""
 
-    def __init__(self):
-        super().__init__()
-        self._executor = ProcessPoolExecutor(Config.job_config.nb_of_workers or 1)
+    def __init__(self, scheduler: _AbstractScheduler):
```

Comment:
Side remark for the futur: 
The dispatcher does not really need a scheduler. It only needs the queue. For now, it is good as it is, but in the future, we may want to remove the scheduler dependency here, and directly pass a queue object as a parameter of the init method. It could be either an in-memory queue like today or a more complex queue object using for instance RabbitMq.